distributed/comm/core.py

```diff
@@ -214,8 +214,8 @@ async def _():
     async def on_connection(self, comm: Comm, handshake_overrides=None):
         local_info = {**comm.handshake_info(), **(handshake_overrides or {})}
         try:
-            write = await asyncio.wait_for(comm.write(local_info), 1)
```

Comment:
Instead of introducing another config option, would `distributed.comm.timeouts.connect` be also applicable here?

Reply:
I think we still want a timeout on the listener side, but it could be large (we just want to ensure servers can kill stuck connections). `distributed.comm.timeouts.connect` seems fine to me, both the `comm.write` and `comm.read` should complete within that time.